upgrade/1.2/scripts/upgrade/csm-upgrade.sh

```diff
@@ -121,5 +121,9 @@ else
     echo "====> ${state_name} has been completed"
 fi
 
+state_name="POST CSM Upgrade Validation"
```

Comment:
The BGP test in this suite will require the SW_ADMIN_PASSWORD env var to be set.   We'll need to get that as an input from the user.